src/init.cpp

```diff
@@ -333,7 +333,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-loadblock=<file>", _("Imports blocks from external blk000??.dat file") + " " + _("on startup"));
     strUsage += HelpMessageOpt("-maxorphantx=<n>", strprintf(_("Keep at most <n> unconnectable transactions in memory (default: %u)"), DEFAULT_MAX_ORPHAN_TRANSACTIONS));
     strUsage += HelpMessageOpt("-par=<n>", strprintf(_("Set the number of script verification threads (%u to %d, 0 = auto, <0 = leave that many cores free, default: %d)"),
```

Comment:
- This is unrelated to this PR, so should be a separate issue.
- This is unchanged in upstream, so any changes should be sent there too.
- I cannot find documentation for `strprintf` *anywhere*, so I can't figure out what the actual meaning of `%u` is meant to be.